spec/es_spec_helper.rb

```diff
@@ -0,0 +1,27 @@
+require "logstash/devutils/rspec/spec_helper"
+require "ftw"
+require "logstash/plugin"
+require "logstash/json"
+require "stud/try"
+
```

Comment:
Why is this file named ```es_spec_helper.rb``` ? I would rather use a more clear naming in here. for example ```docker_helper.rb``` or simply ```spec_helper.rb```, what do you think?